doc/rst/usingchapel/building.rst

```diff
@@ -131,9 +131,15 @@ Specifically:
   ``/usr/local/`` or ``~/``.  Note that elevated privileges are likely
   to be required for any system-wide installation locations.
 
+  Please note that for any prefix-installed version of Chapel, the
+  meaning of ``$CHPL_HOME`` (e.g., as printed by compiler messages)
+  typically refers to ``/dir/for/install/share/chapel/x.yz`` where
+  ``x.yz`` is the Chapel version that was installed.
```

Comment:
I'm not sure what "typically" is adding here. In particular, I think it might lead one to believe that it depends on the OS or which path was chosen or the version of Chapel.

Is the idea that `CHPL_HOME` could be set with an environment variable while using such an installed `chpl` ? I think you will get a warning if you do that & at the very least I would discourage it in the documentation.

Or were you thinking of some other situation that caused you to write "typically"?

Reply:
I wasn't entirely happy with using "typically" here either, but leaned on it as a wiggle word in case someone came across `$CHPL_HOME/bin/...` or `$CHPL_HOME/lib/...` in some context, where this guidance wouldn't apply.  On one hand, I felt nervous removing the qualifier in case such cases came up; at the same time, I wasn't convinced I wanted (or needed) to get into explaining the exceptions...

Reply:
(PS — I'm going to wait to see what you think about this Michael before proceeding even though you've approved the PR).

Reply:
It is OK with me to proceed with "typically" (although, for me it also begs the question, what are the exceptions). Either way what you have here is an improvement over the docs we had in that it explains the usual situation at all.

> in case someone came across ... where this guidance wouldn't apply

if that happened, we would learn what the specific situation where it is wrong, and then could update the documentation!

Even so, I don't think it's worth fussing over it further for this release.